x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.test.ts

```diff
@@ -1127,6 +1240,18 @@ describe('Jira service', () => {
       );
     });
 
+    test('it should throw an error when only one double quote is used', async () => {
```

Comment:
This test does not test if the double quote is used because we throw the error. There is already a test for checking the errors for the `getIssues` function. Let's convert this test to check if it respects the `errorMessage` property.

Reply:
fixed in [78b037e](https://github.com/elastic/kibana/pull/145610/commits/78b037ede18633b4c8175dacfccae5f03eacf691)